sonar-project.properties

```diff
@@ -0,0 +1,6 @@
+sonar.projectKey=LedgerHQ_ledger-live_AYoXRskslck2EnGRKg2f
+sonar.sources=apps/ledger-live-desktop/src/
+sonar.exclusions=**/*.java,**/node_modules/,**/.turbo,**/.pnpm-*,**/temp,**/.eslintcache,**/tsconfig.tsbuildinfo,**/*.png,**/*.env
+sonar.coverage.exclusions=**/*.java
+sonar.tests=apps/ledger-live-desktop/tests/
```

Comment:
This is actually Playwright folder. It should only contain e2e tests. Unit tests are in the src folder.